ferrilata-devilline/Controllers/PitchController.cs

```diff
@@ -0,0 +1,24 @@
+﻿using System.Linq;
+using ferrilata_devilline.Models;
+using Microsoft.AspNetCore.Mvc;
+
+namespace ferrilata_devilline.Controllers
+{
+    public class PitchController : Controller
+    {
+        [Route("api/pitches")]
+        public ActionResult Return_Pitches()
+        {
+            var re = Request;
+
+            var headers = re.Headers;
+            if (headers.ContainsKey("Authorization") && headers["Authorization"].ToString().Length != 0)
+            {
+                return Json(new Pitches());
+            }
+
+            Response.StatusCode = 401;
+            return Json(new Error("Unauthorizied"));
```

Comment:
typo in unauthorized

Reply:
Fixed.
But it's literally a part of the acceptance criteria  :)
![Jira Screenshot](https://i.imgur.com/NFEgo7Q.png)